### ISSUE TITLE
Why no `act` here?

### ISSUE THREAD

**Issue by Naruto-Sasuke** [ZZUTK/SRNTT#6]
https://github.com/ZZUTK/SRNTT/blob/7b2178e62cb95d190786ae9cd6dac6385238ceb1/SRNTT/model.py#L217

It seems more reasonable here if add `act=tf.nn.relu`

**Comment by hcl14** [comment]
I've seen a couple of implementations  of Image-to-Image convolutional networks where last couple of layers were linear (like SRResNet-like architectures after skip-connection). I suppose this can be treated as something like linear selection of the generated features or interpolation between original and enhanced image. 
You can try leaky relu non-linearity there in order to have good gradient, as relu might saturate on some values
